holidayextras/ui-toolkit
# UXUI-239 - Rename some files and remove styles from components and place into docs

#### What does this PR do? (please provide any background)
- We originally decided to add some margin to some of our components when two of the same component are next to one another, however in practical use this causes more headaches than it should, so we have removed the styles from the component and just added them to the docs.
- I have renames `justified_button` component to sit with other components using a `-`

#### What tests does this PR have?
None

#### How can be this tested?
- Check the code for anti-patterns
- run `grunt docs` and check thats the example work as intended (button, flag and justified button)

#### Screenshots / Screencast
<img width="843" alt="screen shot 2015-07-21 at 13 57 13" src="https://cloud.githubusercontent.com/assets/778942/8801162/6a0f5132-2fb0-11e5-9cc0-9049d91ea8d2.png">

#### What gif best describes how you feel about this work?
![css](http://media.giphy.com/media/5pxnxdzdZfXFK/giphy.gif)
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
